### ISSUE TITLE
OPTIFABRIC AND FABRIC API

### ISSUE THREAD

**Issue by Kriish-04** [Chocohead/OptiFabric#1139]
Optifabric stopped working with fabric api and thus I cant put any mods which use fabric api. The reason is since the latest version supports both 1.20 and 1.19 that is it is backwards compatible it clashes with fabric api which only supports 1.20 and thus whenever i launch the game with both optifabric anf fabric api the game crashes with error code : 1 which is incompatible mods. Pls fix this issue by either making the 1.20 version only compatible with 1.20 or any other means possible. Thank You.

**Comment by topia** [comment]
I don't think your assumption is correct. Please upload the latest.log to triage your problem.
If you use fabric API 0.86.0 or later, please downgrade it. (see https://github.com/Chocohead/OptiFabric/issues/1123)

**Comment by Kriish-04** [comment]
> I don't think your assumption is correct. Please upload the latest.log to triage. If you use fabric API 0.86.0 or later, please downgrade it. (see #1123)

Yeah I actually downgraded it to 0.85.0 and now it works but now there is still one problem. It still crashes if I use replaymod otherwise without the mod it works fine. I have attached the log file after the game crashed when I tried to run it with replaymod.
[latest.log](https://github.com/Chocohead/OptiFabric/files/12147349/latest.log)

**Comment by topia** [comment]
Ah, replay mod compatibility is frequently reported here too. Unfortunately, optifabric doesn't have enough developers to resolve issues; they all are busy.

**Comment by Kriish-04** [comment]
Oof I actually needed replaymod for my videos. I guess whenever I need to record them I would have to remove optifabric and optifine and use iris for shaders. Well lets hope this issue gets resolved quickly.

**Comment by AuroraDoesCode** [comment]
> Oof I actually needed replaymod for my videos. I guess whenever I need to record them I would have to remove optifabric and optifine and use iris for shaders. Well lets hope this issue gets resolved quickly.

No guarantee on how soon it would be fixed, optifabric depends on just one person mainly, which is Choco, other fixes tend to be supplied by third party users.
